src/Microsoft.DotNet.Darc/src/Darc/Options/GatherDropCommandLineOptions.cs

```diff
@@ -14,6 +14,9 @@ internal class GatherDropCommandLineOptions : CommandLineOptions
         [Option('i', "id", HelpText = "BAR ID of build.")]
         public int RootBuildId { get; set; }
 
+        [Option("ids", Separator = ',', HelpText = "BAR ID's of the root build that we want to gather. comma separated.")]
```

Comment:
NIT: I never like when there are two parameters that only differ slightly, like a letter. In this case "id" and "idS". Perhaps could rename this to "root-ids" or something similar..

Reply:
Could you instead just allow multiple `--id` arguments? (I don't know if however args are being parsed allows this.)

Reply:
> multiple `--id` arguments?

Looks like that's not supported. https://github.com/commandlineparser/commandline/issues/357 😞

Reply:
> NIT: I never like when there are two parameters that only differ slightly, like a letter. In this case "id" and "idS". Perhaps could rename this to "root-ids" or something similar..

About this suggestion, I think adding an arbitrary prefix to only one version of the parameter would be worse than similar-looking parameter names. I would prefer if `--id` simply took multiple args if there's a real concern about plural vs. singular being hard to read.

Reply:
I can just change it to --id.